docker/rbac.providers.next.Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+# -----------------------------------------------------------------------------
+# Begin base docker image config for Hyperledger RBAC Next Directory
+# This should remain the same for all python containers to maximize caching
+# -----------------------------------------------------------------------------
+FROM hyperledger/sawtooth-validator:1.0
```

Comment:
I believe the sawtooth-validator image supplies the features NEXT needs to instrument consensus. However, I am still trying to understand why these other modules need those features. We could probably find an alternative base image having only the stuff we need. Awaiting feedback in #708.

Reply:
I agree, the python image we use in `rbac-providers-azure` is pretty light, and we can use a lighter weight alpine version since we don't have to use `apt-get` or any other non-pip package manager?

Reply:
Created #726.